### PR TITLE
OSDOCS#11234: Release note

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.17

Issue:
[OSDOCS-11234](https://issues.redhat.com/browse/OSDOCS-11234)

Link to docs preview:
- [Installation and update](https://82374--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-installation-and-update_release-notes)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@jeffdyoung 
Could you please review this PR?
Thanks!

**Comment by sr1kar99** [comment]
@aleskandro 
Could you please review this PR?
Thanks!

**Review by aleskandro — COMMENTED** [review]
/label qe-approved

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,20 @@ With this release, you can now use DNF to install additional packages to your cu
...
 === Installation and update
 
+[id="ocp-4-17-installing-a-cluster-with-multiarch-support_{context}"]
+==== Installing a cluster with the support for configuring multi-architecture compute machines
+
+With this release, you can install an {aws-first} cluster and {gcp-first} cluster with the support for configuring multi-architecture compute machines. After installing the cluster, you can add multi-architecture compute machines to the cluster in the following ways:
+
+* Adding 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines and already includes 64-bit ARM compute machines. In this case, 64-bit x86 is considered the secondary architecture.
+* Adding 64-bit ARM compute machines to a cluster that uses 64-bit x86 control plane machines and already includes 64-bit x86 compute machines. In this case, 64-bit ARM is considered the secondary architecture.
+
```
Comment (jeffdyoung) [inline release_notes/ocp-4-17-release-notes.adoc:69]: In 4.17 we added the ability to deploy an arm64 control planes with x86 workers at install time (and vice versa):
 arm64 cp + x86 workers 
 and 
 x86 cp + arm64 workers.

Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:80]: Updated as follows:

* Adding 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines.
* Adding 64-bit ARM compute machines to a cluster that uses 64-bit x86 control plane machines.

Comment (jeffdyoung) [reply release_notes/ocp-4-17-release-notes.adoc:80]: @sr1kar99 not quite.. we already support adding multi-arch nodes to existing clusters. We now support installing a cluster with a controlplane and workers of different architectures. 
Something like: 
```
With this release, you can install an {aws-first} cluster and {gcp-first} cluster with the support for configuring multi-architecture compute machines. At install time you can specifiy different cpu architecures for the control plane and worker machines  in the following ways:

*  64-bit x86 compute machines and  64-bit ARM control plane machines.
*  64-bit ARM compute machines  and 64-bit x86 control plane machines.```

Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc:80]: Hi @jeffdyoung 
Got it! I missed this point.
Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -70,6 +70,19 @@ With this update, the user-defined labels and tags for {gcp-first} is Generally
 
 For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing user-defined labels and tags for GCP].
 
+[id="ocp-4-17-installing-a-cluster-with-multiarch-support_{context}"]
+==== Installing a cluster with the support for configuring multi-architecture compute machines
+
+With this release, you can install an {aws-first} cluster and {gcp-first} cluster with the support for configuring multi-architecture compute machines. While installing the cluster, you can specifiy different CPU architecures for the control plane and worker machines in the following ways:
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:76]:
```suggestion
With this release, you can install an {aws-first} cluster and {gcp-first} cluster with the support for configuring multi-architecture compute machines. While installing the cluster, you can specify different CPU architectures for the control plane and compute machines in the following ways:
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -70,6 +70,19 @@ With this update, the user-defined labels and tags for {gcp-first} is Generally
...
+
+* 64-bit x86 compute machines and 64-bit ARM control plane machines.
+* 64-bit ARM compute machines and 64-bit x86 control plane machines.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:79]:
```suggestion
* 64-bit x86 compute machines and 64-bit ARM control plane machines
* 64-bit ARM compute machines and 64-bit x86 control plane machines
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple tiny things to address here

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-17-release-notes.adoc (hunk 2 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc]: Updated as per the suggestion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 3 above)
Comment (sr1kar99) [reply release_notes/ocp-4-17-release-notes.adoc]: Updated as per the suggestion.

**Review by skopacz1 — APPROVED** [review]
LGTM

**Pull request merged**